.github/workflows/molecule_tests.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
Magma's github workflows don't have a copyright notice, but probably should.  If you are going to put one up here, lets use the template from other files below (Copyright 2021 The Magma Authors...).